ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimantMapperTest.java

```diff
@@ -30,10 +35,11 @@ public void shouldMapIndividualToCCD() {
         Party party = SampleParty.builder().individual();
 
         //when
-        CCDClaimant ccdParty = claimantMapper.to(party);
+        CCDClaimant ccdParty = claimantMapper.to(party, claim);
 
         //then
         assertThat(party).isEqualTo(ccdParty);
+        Assertions.assertThat(ccdParty.getPartyEmail()).isEqualTo(claim.getSubmitterEmail());
```

Comment:
Unnecessary use of fully qualified name 'Assertions.assertThat' due to existing static import 'uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat'